client/src/game/ui/HanabiCard.ts

```diff
@@ -687,7 +687,7 @@ export default class HanabiCard extends Konva.Group implements NodeWithTooltip {
 
   animateToDeck() {
     const layoutChild = this.parent as unknown as LayoutChild;
-    if (!layoutChild || !layoutChild.parent) {
+    if (layoutChild === undefined || !layoutChild.parent) {
```

Comment:
should this be
if (layoutChild === undefined || layoutChild.parent === undefined) {

Reply:
if (layoutChild === undefined || layoutChild === null || layoutChild.parent === undefined || layoutChild.parent === null) {

Reply:
parent can be null or undefined, i think.  I can add another condition here, but the linter allowed this one...